horovod/common/operations.cc

```diff
@@ -504,6 +504,14 @@ void BackgroundThreadLoop(HorovodGlobalState& state) {
            "allgather and hierarchical allreduce.";
   }
 
+  // Set flag to enable use of batched memcopy kernel on GPU
+  auto horovod_batch_d2d_memcopies =
+      std::getenv(HOROVOD_BATCH_D2D_MEMCOPIES);
+  if (horovod_batch_d2d_memcopies != nullptr &&
+      std::strtol(horovod_batch_d2d_memcopies, nullptr, 10) > 0) {
+    state.batch_d2d_memcopies = true;
```

Comment:
Since the default is now `true`, we should also change this behavior such that if the user specifies `HOROVOD_BATCH_D2D_MEMCOPIES=0` it sets the value to `false`.

Reply:
Yep, good catch. Just fixed this.